master/master.cfg

```diff
@@ -809,11 +816,13 @@ def create_make_factory(os, llvm_branch):
                 factory.addStep(
                     FileUpload(workersrc='distrib/halide.tgz',
```

Comment:
Are we sure file upload should stop performance tests from running? It might be slow and I don't see how it could use much CPU.